scripts/coreos-oemid

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# Usage: coreos-oemid <input image> <output image> OEMID
+# Example: coreos-oemid rhcos.qcow2 rhcos-aws.qcow2 ec2
+# This will add the coreos.oem.id=ec2 to the bootloader arguments.
+# Intended to be used for Ignition.
+
+
+src=$1
+dest=$2
+oemid=$3
+
+fatal() {
+    echo "error: $@" 1>&2
+    exit 1
+}
+
+tmpd=$(mktemp -d /tmp/qcow2-to-vagrant.XXXXXX)
+tmp_dest=${tmpd}/box.img
+cp --reflink=auto ${src} ${tmp_dest}
+
+# http://libguestfs.org/guestfish.1.html#using-remote-control-robustly-from-shell-scripts
+guestfish[0]="guestfish"
+guestfish[1]="--listen"
+guestfish[3]="-a"
+guestfish[4]="${tmp_dest}"
+
+GUESTFISH_PID=
+eval $("${guestfish[@]}")
+if [ -z "$GUESTFISH_PID" ]; then
+    fatal "guestfish didn't start up, see error messages above"
+fi
+cleanup_guestfish () {
+    guestfish --remote -- exit >/dev/null 2>&1 ||:
+}
+trap cleanup_guestfish EXIT ERR
+
+gf() {
+    guestfish --remote -- "$@"
+}
+
+gf run
+gf list-filesystems > ${tmpd}/filesystems.txt
+vg=/dev/coreos/root
+if ! grep -qFe "${vg}" ${tmpd}/filesystems.txt; then
+    sed -e 's,^,# ,' < ${tmpd}/filesystems.txt
+    fatal "Missing LVM VG ${vg} in filesystems"
+fi
+gf mount "${vg}" /
+gf mount "/dev/sda1" /boot
+stateroot=/ostree/deploy/$(gf ls /ostree/deploy)
+rootdir=${stateroot}/deploy/$(gf ls ${stateroot}/deploy | grep -v \.origin)
```

Comment:
Wait, why do we need `stateroot` and `rootdir`?

Reply:
We don't.  I kept it since it's convenient if we need to change something outside of `/boot`, but I can just comment it out for now.